src/Containers/RightSideBar/EditImage.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from '@Inputs/Button';
+
+// API KEY: AIzaSyBW6afsWt6KM6yvzXgfJ9UmUGImHvedZbc
```

Comment:
What is this for? API keys should not be posted publicly, but also I don't know why we need an API key for this